src/squlearn/kernel/matrix/projected_quantum_kernel.py

```diff
@@ -365,6 +365,7 @@ def __init__(
 
         # Set-up of the outer kernel
         if isinstance(outer_kernel, str):
+            kwargs.pop("num_qubits", None)
```

Comment:
I'm only happy with this, if we know why num_qubits is part of the kwargs. Otherwise we have to pop every new additional parameter in the future.

Reply:
Opened new issue #111 to investigate this further so we can merge this